docs/multicluster/user-guide.md

```diff
@@ -198,52 +198,47 @@ In all clusters, a `ClusterSet` CR must be created to define the ClusterSet, and
 two `ClusterClaim` CRs must be created to claim the ClusterSet and claim the
 cluster is a member of the ClusterSet.
 
-- Create `ClusterClaim` and `ClusterSet` in member cluster `test-cluster-east`
-with the following YAML manifest (you can also refer to
-[multicluster_membercluster_template.yaml](../../multicluster/config/samples/clusterset_init/multicluster_membercluster_template.yaml)):
+- Create `ClusterClaim` and `ClusterSet` in the leader cluster
+`test-cluster-north` with the following YAML manifest (you can also refer to
+[multicluster_clusterset_template.yaml](../../multicluster/config/samples/clusterset_init/multicluster_clusterset_template.yaml)):
 
 ```yaml
 apiVersion: multicluster.crd.antrea.io/v1alpha2
 kind: ClusterClaim
 metadata:
   name: id.k8s.io
-  namespace: kube-system
-value: test-cluster-east
+  namespace: antrea-multicluster
+value: test-cluster-north
 ---
 apiVersion: multicluster.crd.antrea.io/v1alpha2
 kind: ClusterClaim
 metadata:
   name: clusterset.k8s.io
-  namespace: kube-system
+  namespace: antrea-multicluster
 value: test-clusterset
 ---
 apiVersion: multicluster.crd.antrea.io/v1alpha1
 kind: ClusterSet
 metadata:
   name: test-clusterset
-  namespace: kube-system
+  namespace: antrea-multicluster
 spec:
   leaders:
     - clusterID: test-cluster-north
-      secret: "member-east-access-token"
-      server: "https://172.18.0.1:6443"
-  members:
-    - clusterID: test-cluster-east
   namespace: antrea-multicluster
```

Comment:
The line "In the leader cluster, the `ClusterSet` spec should include all member clusters
of the ClusterSet." should be removed or move after here with a minor change like below?
"In the leader cluster, the `ClusterSet` spec will eventually include all member clusters
of the ClusterSet."

Reply:
Removed the line.